en/sensor/rangefinders.md

```diff
@@ -78,15 +78,31 @@ The uLanding Radar is not present in "most" firmware by default and must be star
 The *LeddarOne* must typically be added to firmware before it can be used. For setup/usage information see: [LeddarOne](../sensor/leddar_one.md).
 
 
-### TFmini 
+### TFmini
 
 The [Benewake TFmini LiDAR](http://www.benewake.com/en/tfmini.html) is a tiny, low cost, and low power LIDAR with 12m range. 
 
-The sensor should be connected to the SERIAL 4/5 port on Pixhawk devices. It is enabled by setting parameter [SENS_EN_TFMINI](../advanced_config/parameter_reference.md#SENS_EN_TFMINI) > 0.
+> **Note** The lidar driver is currently in master, and should appear in releases from PX4 v1.8.

```

Comment:
@ChristophTobler Is this correct? ie. is this already "Live" in PX4 v1.7? If so I would delete this note.